CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[3.1.4](https://github.com/CFIA-NCFAD/nf-flu/releases/tag/3.1.4)] - 2023-05-12
+
+Update Clair3 to version 1.0.1, option `--clair3_user_variant_model` allows to provide correct variant model which is not included with Clair3.
```

Comment:
Change to:

This release addresses #11 adding a new option, `--clair3_user_variant_model <PATH TO CLAIR3 MODEL>`, to allow user to provide a Clair3 model not included with Clair3, e.g. a [Rerio](https://github.com/nanoporetech/rerio) Clair3 model for r10 flowcells.